target/linux/ath79/image/nand.mk

```diff
@@ -0,0 +1,29 @@
+define Device/glinet_gl-ar300m-common
```

Comment:
Maybe call that one glinet_gl-ar300m-common-nand, as we have glinet_gl-ar300m-common-nor in generic ...

Reply:
Good point.